<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandDialog</class>
 <widget class="QWidget" name="CommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBoxHours">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> hr</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="pushButtonReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditComment</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditCommand">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="pushButtonCurrent">
     <property name="text">
      <string>Now</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QSpinBox" name="spinBoxMinutes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="suffix">
      <string> min</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Co&amp;mmand:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditCommand</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditComment">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Run at</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Show OSD notification</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditComment</tabstop>
  <tabstop>lineEditCommand</tabstop>
  <tabstop>spinBoxHours</tabstop>
  <tabstop>spinBoxMinutes</tabstop>
  <tabstop>pushButtonCurrent</tabstop>
  <tabstop>pushButtonReset</tabstop>
  <tabstop>checkBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
